census/controllers/census.js

```diff
@@ -219,6 +221,39 @@ var submit = function(req, res) {
   });
 };
 
+let _getDiscussionURL = function(req, dataset, place) {
+  /*
+    If `submission_discussion_url` is defined in settings and it is in the
+    format: https://discuss.okfn.org/c/<topic>/<subtopic>, return a new topic
+    url with a prepopulated topic for place and dataset. Otherwise, return the
+    original `submission_discussion_url` without modification. If
+    `submission_discussion_url` is undefined return an empty string.
+  */
+  let submissionDiscussionURL =
+    _.get(req.params.site.settings, 'submission_discussion_url', '');
+  let parsedURL = url.parse(submissionDiscussionURL);
+  // URL is a discourse link
+  if (parsedURL.hostname === 'discuss.okfn.org') {
+    let splitPathName = _.trimLeft(parsedURL.pathname, '/').split('/');
+    // URL is a category link
+    if (splitPathName[0] === 'c') {
```

Comment:
Since you don't have any `else` implementation, I'd say to start with only the Discourse instance base URL and start building up on it. As I said, if we need to scale to multiple platforms / instances then that's another issue and it's probably going to be done differently anyway.

Does it make sense?